virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -26,7 +26,7 @@ You can install {VirtProductName} on a single node cluster, also known as xref:.
 
 If you install your cluster in xref:../../installing/installing-fips.adoc#installing-fips-mode_installing-fips[FIPS mode], no additional setup is required for {VirtProductName}.
 
-include::modules/virt-hardware-os-requirements.adoc[leveloffset=+1]
+* If your cluster uses worker nodes with different CPUs, live migration failures can occur because different CPUs have different capabilities. To avoid such failures, use CPUs with appropriate capacity for each node and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
```

Comment:
Did you intend to make this change? I added this module in a recently merged PR.

Reply:
Must have happened during the merge conflict for this file. I re-added the include.